src/CLRunner.ts

```diff
@@ -378,12 +482,12 @@ export class CLRunner {
         const conversationReference = BB.TurnContext.getConversationReference(activity)
 
         // Validate request
-        if (activity.from?.id === undefined) {
+        if (activity.from ?.id === undefined) {
```

Comment:
>  [](start = 25, length = 1)

Is this a new style to have space here?

Reply:
Not sure where these extra spaces came from.  Looks like editor added them